### ISSUE TITLE
Accessibility 

### ISSUE THREAD

**Issue by IeuanWalker** [LuckyDucko/Mopups#15]
## Screenreader 

In RG.Popups a workaround was added to prevent the screenreader from working on the background page.
```csharp
 static void HandleAccessibilityWorkaround(PopupPage page)
            {
                if (page.AndroidTalkbackAccessibilityWorkaround)
                {
                    var navCount = XApplication.Current.MainPage.Navigation.NavigationStack.Count;
                    var modalCount = XApplication.Current.MainPage.Navigation.ModalStack.Count;
                    XApplication.Current.MainPage.GetOrCreateRenderer().View.ImportantForAccessibility = ImportantForAccessibility.NoHideDescendants;

                    if (navCount > 0)
                    {
                        XApplication.Current.MainPage.Navigation.NavigationStack[navCount - 1].GetOrCreateRenderer().View.ImportantForAccessibility = ImportantForAccessibility.NoHideDescendants;
                    }
                    if (modalCount > 0)
                    {
                        XApplication.Current.MainPage.Navigation.ModalStack[modalCount - 1].GetOrCreateRenderer().View.ImportantForAccessibility = ImportantForAccessibility.NoHideDescendants;
                    }

                    DisableFocusableInTouchMode(XApplication.Current.MainPage.GetOrCreateRenderer().View.Parent);
                }
            }

            static void DisableFocusableInTouchMode(IViewParent? parent)
            {
                var view = parent;
                string className = $"{view?.GetType().Name}";

                while (!className.Contains("PlatformRenderer") && view != null)
                {
                    view = view.Parent;
                    className = $"{view?.GetType().Name}";
                }

                if (view is Android.Views.View androidView)
                {
                    androidView.Focusable = false;
                    androidView.FocusableInTouchMode = false;
                }
            }
```
The property `AndroidTalkbackAccessibilityWorkaround` was added, but personally don't think it's needed and the popup should just be accessible by default.

## Keyboard navigation 
This was never fixed but would be nice to make the control fully accessible.
https://github.com/rotorgames/Rg.Plugins.Popup/issues/728

This may help - https://stackoverflow.com/a/64632343/4353708

**Comment by IeuanWalker** [comment]
Have looked into it myself but couldn't work out how to get the `GetOrCreateRenderer()` as it's missing in maui.

**Comment by LuckyDucko** [comment]
>The property AndroidTalkbackAccessibilityWorkaround was added, but personally don't think it's needed and the popup should just be accessible by default.

I added this as I was worried about unintended consequences of the workaround I made However, I agree that it should be accessible by default.

The keyboard navigation thing im hoping is simple as the stack overflow shows, I'll look into it soon. Right now, just updating the AwaitablePopups repo to use this under Mopups.Awaitable, as it is causing me pain in a new MAUI app I am creating to not have my toolset with me. (and then also properly fixing the back navigation and releasing that too!)
